railties/test/generators/app_generator_test.rb

```diff
@@ -56,6 +56,13 @@ class AppGeneratorTest < Rails::Generators::TestCase
   # brings setup, teardown, and some tests
   include SharedGeneratorTests
 
+  def assert_generates_with_bundler(options = {})
+    generator([destination_root], options)
+    assert_called_with(generator, :bundle_command, [ ['install'], ["exec spring binstub --all"] ]) do
+      quietly { generator.invoke_all }
+    end
```

Comment:
We still can't use `assert_called_with` here. And that's fine.

Minitest Mock's expect supports a block which I think means we can selectively make the expect into a stub: https://github.com/seattlerb/minitest/blob/master/lib/minitest/mock.rb#L56